include/radix/core/gl/OpenGL.hpp

```diff
@@ -1,6 +1,10 @@
 #ifndef RADIX_CORE_GL_OPENGL_HPP
 #define RADIX_CORE_GL_OPENGL_HPP
 
+#ifdef _WIN32
+#define RADIX_GL_USE_GLAD
```

Comment:
I can remove the define, I found I can put it in the preprocessor in VS so that works better. Will commit soon.